deis-cleanup.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+require "open3"
+require "csv"
+
+KEEP_LAST_VERSIONS = ENV["KEEP_LAST_VERSIONS"].to_i || 2
+DRY_RUN = ENV["DRY_RUN"] || false
```

Comment:
I would go with integer with `ENV.fetch("DRY_RUN", 0).to_i`. It will give you `0` when a key is not present. On the other hand user will have to remember that he should use `0` or `1` values (or another integer values).

YAML is good enough if we don't need to worry about security in this case. Basically it should not be used to load untrusted data, because it allows to execute some code.